infra/gcb/build.py

```diff
@@ -265,6 +274,13 @@ def get_build_steps(project_yaml, dockerfile_path):
                upload_url,
             ],
           },
+          # upload targets list
+          {'name': 'gcr.io/oss-fuzz-base/uploader',
+           'args': [
+               '/workspace/{0}'.format(TARGETS_LIST_FILENAME),
+               targets_list_url,
```

Comment:
maybe we should do this for libfuzzer + asan build only?

Reply:
add that complicates the logic a bit, but good point. Every sanitizer will now upload their own targets.list.{sanitizer}. This could also be useful to people who need to know if a particular sanitizer is enabled.

Reply:
(the buckets are different for each engine, so we're not overwriting any target list files)

Reply:
Good call!